### DISCUSSION TITLE
Ha su tablet

### DISCUSSION THREAD

**Discussion by merovese** [indomus/forum#2160] [HUB: Home Assistant]
Ciao a tutti, ieri ho dovuto ripristinare il tablet sul quale normalmente è installata l’app ha.
Una cosa strana peró mi è successa, prima del ripristino quando aprivo l’app ha il display non si spegneva mai e mi ricordo di averlo impostato in questo modo) ma dopo il ripristino non trovo più tale opzione sull’app reinstallata da capo, nelle impostazioni del tablet al massimo si può tenere attivo per 10 minuti. Potreste aiutarmi?

**Comment by calz64** [comment]
Ciao per me lo trovi nel menu 'Impostazioni > app complementare'.

**Reply by merovese** [reply]
Macchè, appena visto, non c’è nulla, ma sono straconvinto che c’era una cosa del tipo “mantieni schermo acceso” da qualche parte

**Reply by calz64** [reply]
per me è ancora li.
![Screenshot_20221212-133411_Home Assistant](https://user-images.githubusercontent.com/84615011/207046768-0dfc2ee3-a039-49b4-a1d5-7da7004a50c6.jpg)
![Screenshot_20221212-133637_Home Assistant](https://user-images.githubusercontent.com/84615011/207046926-22d4bb7a-9ec5-4c40-8b67-ee48f93a210b.jpg)

**Reply by merovese** [reply]
Mmmm qualcosa non torna 
![7469768D-42D5-4059-B92A-22CD70A999C8](https://user-images.githubusercontent.com/1671062/207048281-54b3048b-6f23-48dc-b897-12d47e062c02.jpeg)
![BD75EC36-584A-4EAE-8F69-FB249A7354DA](https://user-images.githubusercontent.com/1671062/207048287-c9a91733-fb19-4e02-92be-cb943844d587.jpeg)
![E75D8BC8-A509-453D-861F-C7F9A521C097](https://user-images.githubusercontent.com/1671062/207048289-da271bf4-7ad4-4fe4-8e7d-1f0a87abb3a1.jpeg)

**Reply by calz64** [reply]
forse è 2022.12.3 che è cambiata.

**Comment by merovese** [comment]
Mi sa proprio di sì, ma dai credo che da qualche parte l’abbiano infilata no?
